grakn-migration/csv/src/main/java/ai/grakn/migration/csv/Main.java

```diff
@@ -40,12 +39,15 @@
  */
 public class Main {
 
+    static boolean newClusterManager = false;
+
     public static void main(String[] args) {
         start(null, args);
     }
 
     public static void start(ClusterManager manager, String[] args){
```

Comment:
Yes, this is very stinky.

This is because, in this code, the Migrators use the Loaders which need access to a TaskManager (that comes from the ClusterManager).

When I turn the Loader into a client, this will all disappear.